pcdsdevices/mpod_apalis.py

```diff
@@ -212,10 +248,25 @@ class MPODApalisModule(BaseInterface, GroupDevice):
                  kind='normal', string=True,
                  doc='Clears all MPOD module faults')
 
+    model = Cpt(EpicsSignalRO, ':Model', kind='omitted', string=True,
+                doc="Model number")
+
+    limit_pos = Cpt(EpicsSignalRO, ':VoltageLimit', kind='omitted',
+                    doc='Positive voltage limit as a % of the max voltage')
+
+    limit_neg = Cpt(EpicsSignalRO, ':VoltageLimitNegative', kind='omitted',
+                    doc='Negative voltage limit as a % of the max voltage')
```

Comment:
Apparently these update often (+/- a fraction of a %)

Reply:
Are these not static settings? If they're being calculated/generated then maybe this is also not the right setting to pull

Reply:
Possibly so, if you monitor them they fluctuate slightly